skins/oasis/css/core/ads.scss

```diff
@@ -14,6 +14,10 @@
 	display: none !important;
 }
 
+.source-point .wikia-ad.default-height {
```

Comment:
What are the reasons for this new rule? Is it really needed with the whole JavaScript logic of copying styles between elements?